source/read-write-concern/tests/operation/default-write-concern-3.4.yml

```diff
@@ -0,0 +1,113 @@
+# Test that setting a default write concern does not add a write concern
+# to the command sent over the wire.
+# Test operations that require 3.4+ server, where all commands started
+# to accept a write concern.
+
+data:
+  - {_id: 1, x: 11}
+  - {_id: 2, x: 22}
+collection_name: &collection_name default_write_concern_coll
+database_name: &database_name default_write_concern_db
+other_collection_name: &other_collection_name other_collection_name
+
+runOn:
+    - minServerVersion: "3.4"
+
+tests:
+  - description: Aggregate with $out omits default write concern
+    operations:
+      - object: collection
+        collectionOptions: {writeConcern: {}}
+        name: aggregate
+        arguments:
+          pipeline: &out_pipeline
+            - $match: {_id: {$gt: 1}}
+            - $out: *other_collection_name
+          batchSize: 2
```

Comment:
I don't think `batchSize` is relevant when using `$out`, so perhaps this can be deleted. It's also not mentioned in your APM assertion.

Likewise for the other aggregate tests later in this file. The `$merge` test in the other file doesn't use `batchSize`, so that's fine.

Reply:
Yeah this was copy-pasta from aggregate-out.yml in CRUD. Removed.